cli/src/test/java/com/devonfw/tools/ide/commandlet/InstallPluginCommandletTest.java

```diff
@@ -0,0 +1,38 @@
+package com.devonfw.tools.ide.commandlet;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeTestContext;
+import com.devonfw.tools.ide.os.SystemInfo;
+import com.devonfw.tools.ide.os.SystemInfoMock;
+
+/**
+ * Integration test of {@link InstallPluginCommandlet}.
+ */
+public class InstallPluginCommandletTest extends AbstractIdeContextTest {
+
+  /**
+   * Tests if {@link InstallPluginCommandlet} can install intellij plugins with custom url.
+   *
+   * @param os String of the OS to use.
+   */
+  @ParameterizedTest
+  @ValueSource(strings = { "windows", "mac", "linux" })
+  public void testInstallPlugin(String os) {
+
+    // arrange
+    IdeTestContext context = newContext("intellij");
+    SystemInfo systemInfo = SystemInfoMock.of(os);
+    context.setSystemInfo(systemInfo);
+    InstallPluginCommandlet commandlet = new InstallPluginCommandlet(context);
+    commandlet.tool.setValueAsString("intellij", context);
+    commandlet.plugin.setValueAsString("ActivePlugin", context);
```

Comment:
Awesome that you also add new tests for tool commandlets not yet covered even though already implemented before (by someone else) 👍 
A nice improvement would be to install the inactive plugin since this is the more interesting use-case for this scenario:
https://github.com/devonfw/IDEasy/blob/main/cli/src/test/resources/ide-projects/intellij/project/settings/intellij/plugins/InactivePlugin.properties